src/Webhook/WebhookEventHandlerChain.php

```diff
@@ -24,7 +24,7 @@
 final readonly class WebhookEventHandlerChain
 {
     /**
-     * @var list<WebhookHandlerInterface>
+     * @var array<WebhookHandlerInterface>
```

Comment:
```suggestion
     * @var list<WebhookHandlerInterface>
```
Lets keep this and add it to the baseline instead.

Reply:
Actually i think WebhookHandlerInterface[] is already implying its a list.

Reply:
can you help me understand why we need to keep `list` and add it to baseline? Is this an issue with phpstan?

Reply:
It looks like a false positive to me

Reply:
but `iterator_to_array` could actually preserve the keys of the `$handlers` array, so `list` I think is inaccurate in this case.

We can solve this if we explicitly don't preserve the keys:

```php
$this->handlers = iterator_to_array($handlers, false);
```

What do you think?

Reply:
We know 100% that there is a list coming with numeric ascending keys. So list is still the right type here

Reply:
@silasjoisten ok so if we set `preserve_keys` to `false` it should be valid, and this would also make the annotation valid and phpstan happy

Reply:
Let's do it

Reply:
Can you please also release a 0.3 afterwards? Many thanks

Reply:
sure